tests/frameworks/layers/l2l4_network_creation_test.py

```diff
@@ -193,6 +193,52 @@ def testL4L2ColumnCreate(self):
     net.run(3)
 
 
+  def testL4L2ColumnLinks(self):
+    """
+    In this simplistic test we create a network and ensure that it has the
+    correct links between regions.
+    """
+
+    # Create a simple network to check its architecture
+    net = createNetwork(networkConfig1)
+
+    links = net.getLinks()
+
+    # Make sure that we have the right number before going on to specifics
+    self.assertEqual(len(list(net.getLinks())), 6, "Incorrect number of links")
+
+    # These are all the links we're hoping to find
+    desired_links=[("sensorInput_0", "dataOut", "L4Column_0", "activeColumns"),
+      ("L2Column_0", "feedForwardOutput", "L4Column_0", "apicalInput"),
+      ("externalInput_0", "dataOut", "L4Column_0", "basalInput"),
+      ("L4Column_0", "predictedActiveCells", "L2Column_0",
+       "feedforwardGrowthCandidates"),
+      ("L4Column_0", "activeCells", "L2Column_0", "feedforwardInput"),
+      ("sensorInput_0", "resetOut", "L2Column_0", "resetIn")]
+
+    # This gets the actual C-based link objects and discards their tags
+    links = map(lambda x: x.second, links)
```

Comment:
Use a list comprehension instead of `map`. `map` should pretty much never be used in Python